mail_activity_team/tests/test_mail_activity_team.py

```diff
@@ -2,6 +2,8 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from odoo.tests.common import TransactionCase
 from odoo.exceptions import ValidationError
+from odoo.fields import Datetime
+from datetime import timedelta
 
 
 class TestMailActivityTeam(TransactionCase):
```

Comment:
Could you replace TransactioCase w/ SavepointCase?
All these records are expensive (especially employees and users) and it would save quite some time in CI.
If you don't have time, please, add a comment TODO and at least use `tracking_disable=True` in ctx  + `no_reset_password=True` :pray:

Reply:
Changed to SavepointCase!